specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/examples/CosmosDBPrivateEndpointConnectionGet.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
```

Comment:
[nit] Examples should resemble real data as closely as possible - please use a dummy guid for "subscriptionId"